warehouse/query-core/src/test/java/datawave/query/planner/AnyFieldQueryPlannerNoFieldExpansionTest.java

```diff
@@ -0,0 +1,337 @@
+import static datawave.query.testframework.RawDataManager.RE_OP;
+import static datawave.query.testframework.RawDataManager.RN_OP;
+
+public class AnyFieldQueryPlannerNoFieldExpansionTest extends AbstractFunctionalQuery {
+    
+    private static final Logger log = Logger.getLogger(AnyFieldQueryPlannerNoFieldExpansionTest.class);
+    
+    @BeforeClass
+    public static void filterSetup() throws Exception {
+        Collection<DataTypeHadoopConfig> dataTypes = new ArrayList<>();
+        FieldConfig generic = new GenericCityFields();
+        generic.addReverseIndexField(CityField.STATE.name());
+        generic.addReverseIndexField(CityField.CONTINENT.name());
+        dataTypes.add(new CitiesDataType(CityEntry.generic, generic));
+        
+        final AccumuloSetupHelper helper = new AccumuloSetupHelper(dataTypes);
+        connector = helper.loadTables(log);
+    }
+    
+    public AnyFieldQueryPlannerNoFieldExpansionTest() {
+        super(CitiesDataType.getManager());
+    }
+    
+    @Test
+    public void testEqual() throws Exception {
+        log.info("------  testEqual  ------");
+        for (final TestCities city : TestCities.values()) {
+            String cityPhrase = EQ_OP + "'" + city.name() + "'";
+            String query = Constants.ANY_FIELD + cityPhrase;
+            String anyCity = Constants.ANY_FIELD + cityPhrase;
+            String plan = getPlan(query, false, true);
+            assertPlanEquals(anyCity, plan);
+        }
+    }
+    
+    @Test
+    public void testNotEqual() throws Exception {
+        log.info("------  testNotEqual  ------");
+        for (final TestCities city : TestCities.values()) {
+            String cityPhrase = " != " + "'" + city.name() + "'";
+            String anyCity = "!(" + Constants.ANY_FIELD + EQ_OP + "'" + city.name() + "')";
+            String query = Constants.ANY_FIELD + cityPhrase;
+            try {
+                String plan = getPlan(query, false, true);
+                Assert.fail("expecting exception");
+            } catch (FullTableScansDisallowedException e) {
+                // expected
+            }
+            
+            this.logic.setFullTableScanEnabled(true);
+            try {
+                String plan = getPlan(query, false, true);
+                assertPlanEquals(anyCity, plan);
+            } finally {
+                this.logic.setFullTableScanEnabled(false);
+            }
+        }
+    }
+    
+    @Test
+    public void testAnd() throws Exception {
+        log.info("------  testAnd  ------");
+        String cont = "europe";
+        for (final TestCities city : TestCities.values()) {
+            String cityPhrase = EQ_OP + "'" + city.name() + "'";
+            String contPhrase = EQ_OP + "'" + cont + "'";
+            String query = Constants.ANY_FIELD + cityPhrase + AND_OP + Constants.ANY_FIELD + contPhrase;
+            String anyQuery = Constants.ANY_FIELD + cityPhrase + " && " + Constants.ANY_FIELD + contPhrase;
```

Comment:
The AND_OP constant can be changed from " AND " to " && ". In the same context OR could also be changed to " || ".

Reply:
That would be cleaner....I will work that

Reply:
Actually that would break the Lucene query test.  I will make a separate constant.

Reply:
making JEXL_AND_OP and JEXL_OR_OP